static/flow/views/landing-page-data-set-view.js

```diff
@@ -245,7 +245,7 @@ var LandingPageDataSetView = function(options) {
         viewProgramButton.click(item, function(e) {
             console.log("[DEBUG] viewProgramButton click", e.data);
             var editor = getTopLevelView('program-editor-view');
-            editor.loadProgramFromSpec({programdata: programdata});
+            editor.loadProgramFromSpec({programdata: programData});
```

Comment:
Thanks for making these values more readable!

Reply:
Yup